features/commands.js

```diff
@@ -306,7 +306,26 @@ Here's an article explaining the difference between the two: https://goshakkk.na
 
         fetchMsg.delete();
       }
-    }
+		},
+		{
```

Comment:
Please fix the indentation on these couple lines to make them consistent

Reply:
Yeah, I'm okay with this.